src/components/Acknowledgement/Acknowledgement.tsx

```diff
@@ -140,8 +156,13 @@ export const WarningAcknowledgement = ({
               style={popoverStyle}
               onAnimationComplete={handleAnimationComplete}
             >
-              <Box as={FiAlertTriangle} color='red.500' size='80px' mb={4} />
+              {CustomIcon ? (
```

Comment:
nitpick: if we want to, we could consume `icon` directly with object destructuring renuming, to avoid declaring `<CustomIcon />` only for the sake of react component capitalization conventions

Reply:
963d000cf0c9153792397ffbf97074a910f34c80